diff --git a/dist/app/components/nuxt-link.js b/dist/app/components/nuxt-link.js
index b46caf9b6c677cc335217abb7a89798a8d65e977..97299e26912d0f64e6006d62ea710c889fa1d3fb 100644
--- a/dist/app/components/nuxt-link.js
+++ b/dist/app/components/nuxt-link.js
@@ -1,4 +1,4 @@
-import { useNuxtApp, useRuntimeConfig } from "../nuxt.js";
+import { tryUseNuxtApp, useNuxtApp, useRuntimeConfig } from "../nuxt.js";
 import { navigationDiagnostics } from "../diagnostics/navigation.js";
 import { encodeRoutePath, navigateTo, resolveRouteObject, useRouter } from "../composables/router.js";
 import { renderDiagnostics } from "../diagnostics/render.js";
